impedance/models/circuits/elements.py

```diff
@@ -331,8 +331,8 @@ def TLMQ(p, f):
     """
     omega = 2 * np.pi * np.array(f)
     Rion, Qs, gamma = p[0], p[1], p[2]
-    Zs = Qs * (1j * omega) ** gamma
-    Z = np.sqrt(Rion / Zs) / np.tanh(np.sqrt(Rion * Zs))
+    Zs = Qs * (1j * omega) ** -gamma
```

Comment:
I don't think this form of `Zs` matches the paper. `Zs = Qs * (1j * omega) ** -gamma` is:
## $Z_s=Qs(j\omega)^{-\gamma}= \frac{Qs}{j\omega^(\gamma)}$

I think this should actually be `Zs = 1 / (Qs * (1j * omega) ** gamma)`, while keeping the equation for `Z` as written here.  Then, the expected results in the test will be unchanged.

Reply:
Yeah, good call. Confused myself since I added parentheses and switched it to a negative exponent, but then removed them... 😵‍💫